style.css

```diff
@@ -427,3 +438,221 @@ textarea::placeholder {
   margin: 0 auto;
   width: 8rem;
 }
+
+@media only screen and (min-width: 768px) {
+  .mobile-menu {
+    display: none;
+  }
+
+  #navigation {
+    display: flex;
+    justify-content: space-between;
+    padding: 16px 66px;
+    box-shadow: 0 1px 10px -5px grey;
+    position: fixed;
+    height: 72px;
+  }
+
+  nav {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .desktop-links {
+    display: flex;
+    justify-content: space-around;
+    width: 300px;
+    align-items: center;
+  }
+
+  .desktop-links li {
+    list-style: none;
+    color: #344563;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    cursor: pointer;
+  }
+
+  .card-tools li {
+    padding: 4px 4px;
+  }
+
+  .language-list li {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .desktop-links li:hover {
+    color: #6070ff;
+  }
+
+  .hero-section {
+    background-image: url(./images/Header\ bg.svg);
+    background-size: 100% 691.46px;
+    background-position-x: right;
+    padding-top: 8rem;
+    padding-bottom: 9rem;
+    height: 846px;
+  }
+
+  .hero-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    width: 70%;
+    margin: auto;
+    margin-top: 100px;
+    gap: 50px;
+  }
+
+  #name {
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    margin: 10px 0;
+    height: 50px;
+  }
+
+  .details {
+    width: 100%;
+    margin-top: 1rem;
+    height: 28px;
+  }
+
+  .connect {
+    transform: translateY(-2rem);
+    margin-top: 1.5rem;
+    height: 0;
+  }
+
+  .hero-section ul {
+    margin-top: -3em;
+  }
+
+  #work-section {
+    display: none;
+  }
+
+  #work-section-desktop {
+    width: 80%;
+    margin: auto;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-auto-flow: row;
+    justify-items: center;
+    align-items: center;
+    margin-top: 46px;
+  }
+
+  .card {
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 36px;
+    max-width: initial;
+  }
```

Comment:
- [ ] Although the design of your card is impressive :+1:, it is essential to maintain consistency in the design of all the cards. Currently, there is a width difference between your first card and the subsequent cards, which does not adhere to the project requirements. By ensuring that all the cards have the same design, you can maintain a cohesive and visually appealing layout. 


https://user-images.githubusercontent.com/105117832/236425569-99d97708-bd79-471d-9fd9-16855d46eeb0.mp4